fsharp-backend/src/LibExecution/DvalReprLegacyExternal.fs

```diff
@@ -1,5 +1,5 @@
 /// Ways of converting Dvals to/from strings, with external use allowed
-module LibExecution.DvalReprExternal
+module LibExecution.DvalReprLegacyExternal
```

Comment:
I think the comments here could start with something like:

"The remaining functions in this file are used by the HttpClient or the Http framework, and cannot be changed in their existing usages. The plan to remove them is to replace both the Http framework with a middleware based design, with some of that middleware being a typed json serializer. The typed json serializer would be a standard library function, not something that's an inate part of the framework/client in the way that these functions are"